src/main/java/com/hisabkitab/entity/Expense.java

```diff
@@ -0,0 +1,112 @@
+package com.hisabkitab.entity;
+
+import com.hisabkitab.entity.enums.SplittingType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import org.joda.time.DateTime;
+
+@Entity
+@Table(name = "expense")
+public class Expense {
+
+    @Id
+    @GeneratedValue(strategy= GenerationType.IDENTITY)
+    private Long id;
+    private String description;
+    private Double Amount;
+    @ManyToOne
+    @JoinColumn(name = "paid_by_id", referencedColumnName = "id")
```

Comment:
In Debt table we have kept this column name as owed_by/owed_to. Should we follow same everywhere?